packages/expo-dev-menu/app/screens/MainScreen.tsx

```diff
@@ -0,0 +1,332 @@
+import {
+  View,
+  Text,
+  Heading,
+  Row,
+  Spacer,
+  XIcon,
+  Button,
+  Divider,
+  ClipboardIcon,
+  DebugIcon,
+  HomeFilledIcon,
+  InspectElementIcon,
+  PerformanceIcon,
+  RefreshIcon,
+  RunIcon,
+  StatusIndicator,
+  Image,
+} from 'expo-dev-client-components';
+import * as React from 'react';
+import { Switch } from 'react-native';
+
+import { useBuildInfo } from '../hooks/useBuildInfo';
+import { useClipboard } from '../hooks/useClipboard';
+import { useDevSettings } from '../hooks/useDevSettings';
+
+export function MainScreen() {
+  const buildInfo = useBuildInfo();
+  const { devSettings, actions } = useDevSettings();
+
+  const urlClipboard = useClipboard();
+  const buildInfoClipboard = useClipboard();
+
+  function onCopyUrlPress() {
+    const { hostUrl } = buildInfo;
+    urlClipboard.onCopyPress({ hostUrl });
+  }
+
+  function onCopyBuildInfoPress() {
+    const { runtimeVersion, sdkVersion, appName, appVersion } = buildInfo;
+    buildInfoClipboard.onCopyPress({ runtimeVersion, sdkVersion, appName, appVersion });
+  }
+
+  const hasCopiedBuildInfoContext = Boolean(buildInfoClipboard.clipboardContent);
```

Comment:
this will reset when the menu is closed and reopened, correct?

Reply:
yes - I think the "context" is a typo (meant to write "content") - its reset after ~3 seconds, and would be reset completely on mount / unmount